src/main/resources/swagger.api/mod-search.yaml

```diff
@@ -25,6 +25,20 @@ paths:
               example: examples/searchResult.sample
               schema:
                 $ref: '#/components/schemas/searchResult'
+        '400':
+          description: Error response when NullPointerException occurred during code execution
```

Comment:
`Error response when NullPointerException occurred during code execution` shouldn't it be a 500 error?